Sources/PasscodeKit/Views/KeypadView.swift

```diff
@@ -6,12 +6,17 @@
 //
 
 import SwiftUI
+import LocalAuthentication
 
 struct KeypadView: View {
+    
+    @Environment(\.passcode) private var passcodeEnv
```

Comment:
Since we only need the `PasscodeManager` here we can simply only access it, instead of the entire passcode environment.

```swift
@Environment(\.passcode.manager) private var passcodeManager
```